examples/with-compiled-css/.babelrc

```diff
@@ -1,4 +1,3 @@
 {
-  "presets": ["@babel/preset-react"],
-  "plugins": ["@babel/plugin-syntax-jsx"]
```

Comment:
the additional plugin is required, for me it doesn't work without it, so `.babelrc` should look like:

```
{
  "presets": ["next/babel"],
  "plugins": ["@babel/plugin-syntax-jsx"]
}
```